README.md

```diff
@@ -32,8 +32,33 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 $ cp .env.template .env  # (first time only)
 ```
 
+As we will be using the Trello API, add the requests library to your list of poetry dependencies in pyproject.toml by running:
+
+'''bash 
+ $ poetry add requests
+'''
```

Comment:
This is useful to record sometimes, but worth being clear that future developers shouldn't need to ever run this again - because the dependency is now listed in your `pyproject.toml`, then they'll get that automatically when they run `poetry install`

Reply:
Removed from ReadMe file